ci/documentation_updater/documentation_updater.py

```diff
@@ -0,0 +1,200 @@
+        exit(1)
+    if args.ignored_files_list and not args.ignored_files_list.is_file():
+        logging.critical("File {} does not exist. Please check '--ignored-files-list' option."
+                         .format(args.ignored_files_list))
+        exit(1)
+    return args
+
+
+def collect_readme(directory, ignored_files=('index.md',)):
+    files = {file.stem: file for file in directory.glob('**/*.md') if file.name not in ignored_files}
+    logging.info('Collected {} readme files'.format(len(files)))
+    if not files:
+        logging.error("No markdown file found in {}. Exceptions - {}. Ensure, that you set right directory."
+                      .format(directory, ignored_files))
+        exit(1)
+    return files
+
+
+def convert(lines):
+    def flatten_links(s):
+        links = re.findall(r"\[.*?\]\(.*?\)", s)
+        for link in links:
+            plain_link = re.sub('[\[\]]', '', link.replace('](', ' (').replace('(', '<').replace(')', '>'))
+            s = s.replace(link, plain_link)
+        return s
+
+    result = ''
+    list_signs = ['-', '*']
+    for line in lines:
+        if len(line.strip()) == 0:
+            result += '\n'
+        elif line.lstrip()[0] in list_signs:
+            result += '\n'
+        if len(result) > 0 and result[len(result)-1] != '\n':
+            result += ' '
+        result += line.rstrip('\n')
+        result = flatten_links(result)
+    result = result.replace("`", "\"").replace("\*", "*")
+    return result.strip()
+
+
+def collect_descriptions(files):
+    descriptions = {}
+    for name, file in files.items():
+        started = False
+        desc = []
+        with open(file, "r", encoding="utf-8") as readme:
+            for line in readme:
+                if line.startswith('##'):
+                    if not started:
+                        started = True
+                        continue
+                    else:
+                        break
+                if started:
+                    desc.append(line)
+        desc = convert(desc)
+        if desc != '':
+            descriptions[name] = desc
+        else:
+            logging.warning('No description found in {} file. '
+                            'Check  compliance with the OMZ Contribution Guide.'.format(file))
+    return descriptions
+
+
+def get_models_from_configs(directory):
+    yaml = ruamel.yaml.YAML()
+    yaml.preserve_quotes = True
+    models = {}
+    model_configs = Path(directory).glob('**/model.yml')
+    for model in model_configs:
+        with open(str(model), "r") as file:
+            models[model.parent.name] = (model, yaml.load(file))
+
+    return models
+
+
+def update_model_descriptions(models, descriptions, mode):
+    update_models = []
+    for name, desc in descriptions.items():
+        model = models.get(name, None)
+        if model is None:
+            logging.warning('For description file {}.md no model found in topologies list'.format(name))
```

Comment:
```suggestion
            logging.warning('For description file {}.md no model found'.format(name))
```

They're called models rather than topologies, and no matter what you call them, there is no list of them in the first place.